splitio/models/telemetry.py

```diff
@@ -650,12 +1225,51 @@ def time(self):
         """
         return self._time
 
+class StreamingEventsAsync(object):
```

Comment:
No need for a `StreamingEventsBase interface here`, right?

Reply:
Nope, there is no shared code or function between threading and asyncio classes